ironfish/src/rpc/routes/accounts/rescanAccount.ts

```diff
@@ -33,6 +34,11 @@ router.register<typeof RescanAccountRequestSchema, RescanAccountResponse>(
     }
 
     if (!scan) {
+      if (node.accounts.updateHeadState) {
+        void node.accounts.updateHeadState.abort()
+        await node.accounts.updateHeadState.wait()
+      }
```

Comment:
If you are running an update head, this will abort abort it so we can start rescanning. A nice usability improvement.